src/Auditable.php

```diff
@@ -131,9 +131,16 @@ protected function auditCreatedAttributes(array &$old, array &$new)
     protected function auditUpdatedAttributes(array &$old, array &$new)
```

Comment:
i fixed my problem for now and maybe some other users, it will be fix in future by someone or i will try to fix it soon.